docs/api.rst

```diff
@@ -231,7 +231,64 @@ Possible Responses
 | 500 Server Error | Internal Raiden node error|
 +------------------+---------------------------+
 
+Token Swaps
+------------
+
+You can perform a token swap by using the token_swap endpoint. A swap consists of two users agreeing on atomically exchanging two different tokens at a particular exchange rate.
+
+By making a ``PUT`` request to ``/api/<version>/token_swaps/<target_address>/<identifier>`` you can either initiate or participate in a token swap with a specific user. The details, along with the role, come as part of the json payload.
+
+Example Request
+^^^^^^^^^^^^^^^
+
+The maker (in our case ``0xbbc5ee8be95683983df67260b0ab033c237bde60``) would do
+
+``PUT /api/1/token_swaps/0x61c808d82a3ac53231750dadc13c777b59310bd9/1337``
+
+with payload
+::
+
+    {
+        "role": "maker",
+        "sending_amount": 42,
+        "sending_token": "0xea674fdde714fd979de3edf0f56aa9716b898ec8",
+        "receiving_amount": 76,
+        "receiving_token": "0x2a65aca4d5fc5b5c859090a6c34d164135398226"
+    }
```

Comment:
Could you add a bit saying that `sending` and `receiving` are from the perspective of each node? Just to be explicit and avoid issues like https://github.com/raiden-network/raiden/issues/379